tests/dualtor/test_ipinip.py

```diff
@@ -126,6 +128,15 @@ def test_decap_standby_tor(
     rand_selected_interface, ptfadapter,
     tbinfo, rand_selected_dut, tunnel_traffic_monitor
 ):
+
+    def verify_downstream_packet_to_server(ptfadapter, port, exp_pkt):
+        """Verify packet is passed downstream to server."""
+        packets = ptfadapter.dataplane.packet_queues[(0, port)]
```

Comment:
Is this where we 'pop out' packtes from the `ptfadapter` queue?

Reply:
We don't need to pop out packets because those packets should be checked and poped by the tunnel traffic utility via those `verify_*` functions defined in `ptf.testutils`.

Reply:
Oh, I see now. So you are saying that when the tunnel traffic monitor uses the `verify_*` methods to check if we received any IP in IP packets, it removes all received packets from the queue, and that is why we need to check server-bound traffic with this `verify_downstream_packet_to_server` method? 

It seems like a design flaw in PTF that verifying packets on one interface could affect the packet queues for a different interface.

Reply:
Yes, ptf retrieves pkts with chronological order, it doesn't care which queue the packet is in. So those `verify_*` functions in the tunnel traffic utility will make ptf drop those packets so that the user could not access them after.

Reply:
I see. Thanks for clarifying!